src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -59,18 +70,51 @@ public void saveTranscriptionRequest(TranscriptionRequestDetails transcriptionRe
             userAccount,
             transcriptionRequestDetails,
             transcriptionStatus,
-            getTranscriptionType(transcriptionRequestDetails.getTranscriptionTypeId()),
+            getTranscriptionTypeById(transcriptionRequestDetails.getTranscriptionTypeId()),
             getTranscriptionUrgencyById(transcriptionRequestDetails.getUrgencyId())
         );
 
         saveTranscriptionWorkflow(
             userAccount,
-            transcriptionRequestDetails,
             transcription,
-            transcriptionStatus
+            transcriptionStatus,
+            transcriptionRequestDetails.getComment()
         );
     }
 
+    @Override
+    @Transactional
+    public UpdateTranscriptionResponse updateTranscription(Integer transcriptionId,
+                                                           UpdateTranscription updateTranscription) {
+
+        TranscriptionEntity transcription = transcriptionRepository.findById(transcriptionId)
+            .orElseThrow(() -> new DartsApiException(TRANSCRIPTION_NOT_FOUND));
+
+        Map<TranscriptionStatusEnum, Set<TranscriptionStatusEnum>> expectedStatuses = new HashMap<>();
+        expectedStatuses.put(AWAITING_AUTHORISATION, Set.of(APPROVED, REJECTED));
```

Comment:
Isn't this part going to be covered by Karens https://tools.hmcts.net/jira/browse/DMP-1128 ticket? There are potentially going to be a few features using this service so not sure it makes sense to focus on the AWAITING_AUTHORISATION status here.

Reply:
Yes it will be refactored to cope with the various `updateTranscription` flows in our design. I just did what I felt I needed to do this sprint for the Approved / Rejected flows.  I've already let Karen know to look against my branch and she is also on this pull request for visibility.  Once this is merged, I'll be looking at [DMP-375](https://tools.hmcts.net/jira/browse/DMP-375) Rejected.